main.js

```diff
@@ -0,0 +1,38 @@
+var button = document.querySelector('button');
+var input = document.querySelector('input');
+var h1 = document.querySelector('h1');
+
+
+
+var insertText = function (){
+var p = document.createElement('p');
+
+var inputValue = input.value;
+p.textContent = "Currently working on..." + inputValue;
+
+ var body = document.querySelector('body');
+   body.appendChild(p);
+
+ };
+
+ button.addEventListener('click', insertText);
+
+
+ // counter
+
+ var number = 25;
+ var intervalID;
+
+
+ var countUp = function(){
```

Comment:
This function says it's going to count up but the function actually counts down. There's a programming concept called "the principle of least surprise."